Homeworks/Homework 6-EC/MSDS_631_Homework_6-EC.ipynb

```diff
@@ -187,21 +184,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
-   "source": []
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
-   "source": []
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Female: Economics\n",
```

Comment:
Can you please change her major to Physics?

Reply:
No, requires registrar approval